dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -254,6 +254,39 @@ public int ResultsPerPrompt
         }
     }
 
+    /// <summary>
+    /// Whether to return log probabilities of the output tokens or not.
+    /// If true, returns the log probabilities of each output token returned in the `content` of `message`.
+    /// </summary>
+    [Experimental("SKEXP0010")]
+    [JsonPropertyName("logprobs")]
+    public bool? EnableLogProbabilities
```

Comment:
Why `bool?` rather than `bool`?

Reply:
I used the same type as in Azure .NET SDK. Also, OpenAI documentation explicitly says that it's bool or null:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/684c9de3-4571-459a-9155-910adc0c634c)

Today, Azure .NET SDK sends always `null`. I thought it's better to keep that behavior rather than start sending `false`, even though it may not change overall behavior.

Reply:
Thanks. Just seems odd to explicitly have three representable states when there's actually only two. It's strange on the OpenAI definition as well, especially with it defaulting to false rather than null.